cmd/krel/cmd/ff.go

```diff
@@ -255,12 +259,19 @@ func runFf(opts *ffOptions) error {
 	if !status.IsClean() {
 		log.Printf("Commit changes:")
 		// TODO: Rewrite using go-git
-		gitAdd := exec.Command("git", "add", "-A")
-		util.Run(gitAdd)
+		if err := command.Execute("git add -A"); err != nil {
+			return err
+		}
 
 		// TODO: Rewrite using go-git
-		gitCommit := exec.Command("git", "commit", "-am", fmt.Sprintf("Results of running update scripts: %s", strings.Join(updateScripts, ",")))
-		util.Run(gitCommit)
+		if err := command.Execute(
+			"git commit -am",
+			fmt.Sprintf(
+				"'Results of running update scripts: %s'",
+				strings.Join(updateScripts, ",")),
```

Comment:
We may add something like `command.Executef(a format, a ...interface{})` for such use cases as well... :thinking: